hpe3par_sdk/client.py

```diff
@@ -3602,16 +3602,7 @@ def _format_srstatld_output(self, out):
         return self.client._format_srstatld_output(out)
 
     def tuneVolume(self, volName, tune_operation, optional=None):
-        info = { 'action': self.TUNE_VOLUME, 'tuneOperation': tune_operation }
-
-        if optional is not None and self.CURRENT_WSAPI_VERSION < self.WSAPI_MIN_VERSION_COMPRESSION_SUPPORT:
-            if 'compression' in optional.keys():
-                del optional['compression']
-        if optional:
-            info = self.client._mergeDict(info, optional)
-        response, body = self.client.http.put(
-            '/volumes/%s' % volName, body=info)
-        return self.getTask(body['taskid'])
+        return Task(self.client.tuneVolume(volName, tune_operation, optional))
```

Comment:
@ansarars Let's not change the signature of the method for return.
You can do as below:
response, body = self.client.tuneVolume(volName, tune_operation, optional)	
return self.getTask(body['taskid'])

Reply:
addressed review comment